src/universal/validation/templates.js

```diff
@@ -77,20 +77,19 @@ export const orgRejectionReason = (value) => value
 
 export const preferredName = (value) => value
   .trim()
-  .required('That\'s not much of a name, is it?')
-  .min(2, 'C\'mon, you call that a name?')
+  .required('That’s not much of a name, is it?')
+  .min(2, 'C’mon, you call that a name?')
   .max(100, 'I want your name, not your life story');
 
 export const task = (value) => value
   .trim()
-  .min(2, 'That doesn\'t seem like much of a task')
+  .min(2, 'That doesn’t seem like much of a task')
   .max(PROJECT_MAX_CHARS, 'Try shortening down the task name');
 
 export const teamName = (value) => value
   .trim()
-  .required('"The nameless wonder" is better than nothing')
-  .min(2, 'The "A Team" had a longer name than that')
-  .max(50, 'That isn\'t very memorable. Maybe shorten it up?');
+  .required('“The nameless wonder” is better than nothing')
```

Comment:
Yeah, if it's important to move away from ", then we should use both “ and ”. not just ”